cinch/playbooks/jenkins_restart_utility.yml

```diff
@@ -0,0 +1,137 @@
+# This playbook aims to gather diagnostics data in the event of
+# trouble with Jenkins, and then restarts Jenkins to keep operations running.
+# By default, only the diagnostics data is collected, but the Jenkins service
+# restart can be enabled by setting the 'restart_jenkins' variable to true:
+
+# Example:
+# ansible-playbook -u root --private-key /path/to/key -i '10.0.183.169', jenkins_restart_utility.yml -e "restart_jenkins=true"
```

Comment:
The service restart functionality requires root as you would expect by default in any distribution.  The diagnostics collection can be done by the jenkins user, except for a single case where root may own the SSH key needed to authenticate to Jenkins to run the "support bundle" collection.  I'll contact you in private for further details.